src/components/Question/Question.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from "react";
+import className from "classnames";
+import "./Question.css";
+
+const answerBox = "answerBox";
+const questionSubmit = "questionSubmit";
+export default class Question extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedAnswer: "",
+      showAnswer: false
+    };
+  }
+
+  handleRadioClick = v => {
+    this.setState({
+      selectedAnswer: v
+    });
+  };
+
+  handleSubmitClick = () => {
+    this.setState({ showAnswer: true });
+    this.props.onHandleQuestionSubmit(
+      this.state.selectedAnswer === this.props.questionToAnswer.a
+    );
+  };
+
+  handleNextClick = () => {
+    this.setState({
+      selectedAnswer: "",
+      showAnswer: false
+    });
+    this.props.onHandleQuestionUpdate(this.props.questionIndex + 1);
+  };
+
+  render() {
+    const arrQuestions = [
+      this.props.questionToAnswer.choice_1,
+      this.props.questionToAnswer.choice_2,
+      this.props.questionToAnswer.choice_3
+    ];
+    let answerRender;
+
+    if (this.state.showAnswer === true) {
+      this.props.questionToAnswer.a === this.state.selectedAnswer
+        ? (answerRender = (
+            <h6 className={className(answerBox, "bg-success")}>
+              You are correct!
+            </h6>
+          ))
+        : (answerRender = (
+            <h6 className={className(answerBox, "bg-danger")}>
+              Incorrect! This is your answer {this.state.selectedAnswer}
+              <br />
+              The correct answer is {this.props.questionToAnswer.a}
+            </h6>
+          ));
+    } else {
+      answerRender = null;
+    }
+
+    const btnDisplay = () => {
```

Comment:
You're repeating a lot of code here. How about just: 

```suggestion
    const btnDisplay = (
       <button
         className={className("btn", "btn-primary", questionSubmit)}
         type="submit"
         onClick={this.handleSubmitClick}
         disabled={!this.state.selectedAnswer}
       >
         Submit
       </button>
     );
```

Reply:
Right......... I keep forgetting that the disabled state is a truth/false value. Thanks!